src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -1020,3 +1020,17 @@ def addTier1Skim(config, skimName, dataTier, primaryDataset, cmsswVersion,
 
     datasetConfig.Tier1Skims.append(skimConfig)
     return
+
+def setStreamMem(config, *streamid, **streamidvalue):
+    """
+    _setStreamMem_
+    For each streamid and streamidvalue in KB, create a dict to update the max
+    memory within the specArgument['Memory']
+    """
+
+    streamidmem = {}
+    for i, a in enumerate(streamid):
+        streamidmem['arg%s' % (i+1)] = a
+    streamidmem.update(streamidvalue)
+    config.Global.StreamMem = streamidmem
```

Comment:
I was thinking that you were going to use our already existing streamer configs. I think this approach could work, but I'm not sure if it is the best to have an extraset of streamer configurations.